tests/sources/projHoist/module/driver_mod.f90

```diff
@@ -0,0 +1,68 @@
+module transformation_module_hoist
+    implicit none
+    integer, parameter :: len = 3
+contains
+
+    subroutine driver(a, b, c)
+        integer, intent(in) :: a
+        integer, intent(inout) :: b(a)
+        integer, intent(inout) :: c(a, len)
+        real :: x, y
+        call kernel1(a, b, c)
+        !    kernel1(a, b, c, kernel1_x, kernel1_y, kernel1_k1_tmp)
```

Comment:
Is this the expected call signature after the transformation?
I would prefer not having this as a comment in the Fortran but rather as a string-comparison in the test to validate also the expected order of the additional arguments, e.g.:
```
assert call.arguments == ('a', 'b', 'c', 'kernel1_x', 'kernel1_y', 'kernel1_k1_tmp')
```

You can rely here on the automatic comparison against strings, which allows tricks like that:
```
>>> [Variable(name='a'), Variable(name='b')] == ['a', 'b']
True
```